### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by svrooij**
This PR will fix 2 small issues:

- TimeZone issue described in #16 
- Not sending the correct headers #32 

For the TimeZones I created a small extension so that it doesn't disturb anything, but it doesn't crash on Ubuntu because of adding duplicate TimeZoneId's

You commented out the sending of the headers with the HttpClient, probably because of that they didn't validate in the `MediaTypeHeaderValue`. I changed it to just send the Content-Type anyway.

### CODE REVIEW

**Comment by sherlock1982** [comment]
I will check, thanks!

**Comment by adz** [comment]
Hi @sherlock1982 - could you consider please merging this and releasing a new nuget?

We have been struggling with code that works with Office365, but fails with Exchange 2010.

Specifically, the error was:

`InnerException was: Microsoft.Exchange.WebServices.Data.ServiceRequestException: 
The request failed. 
Cannot process the message because the content type 'text/plain; charset=utf-8' was not the expected type 'text/xml; charset=utf-8'.

 ---> Microsoft.Exchange.WebServices.Data.EwsHttpClientException: 
Cannot process the message because the content type 'text/plain; charset=utf-8' was not the expected type 'text/xml; charset=utf-8'.`

After this change that allows setting content type, everything appears working ok.
Seems like no content type was set before...

**Comment by svrooij** [comment]
For now I'm just running this in a private nuget feed on Azure DevOps.

**Comment by svrooij** [comment]
Closing this stale PR

**Comment by adz** [comment]
Could we merge instead? Are the issues fixed elsewhere or can I do something to make the PR it acceptable?

**Comment by svrooij** [comment]
@adz they are fixed in this old pr. I'm using a private nuget in the mean time.

Seems this repo isn't maintained anymore.

Yesterday I closed several stale PR of mine

**Comment by svrooij** [comment]
@adz they are fixed in this old pr. I'm using a private nuget in the mean time.

Seems this repo isn't maintained anymore.

Yesterday I closed several stale PR of mine

**Comment by adz** [comment]
Oh right. I thought maybe you were a maintainer. Thanks - I'm also using a fork.

**Comment by tjmoore** [comment]
Could this be merged @sherlock1982 ? I see you've merged a change recently for 2.0.0-beta3 so this repo doesn't seem entirely dead.

Project I'm working on has hit the issue in #16 when running on Linux (CentOS 7 based) and get "An item with the same key has already been added. Key: Dlt/1941" when creating appointments. It would appear that it's this issue.

**Comment by sherlock1982** [comment]
@tjmoore could you pls resolve conflicts? The issue is that i can't really test it - I don't have on premise exchange and don't have a lot of time.
If you say it's ok - let's merge it.

**Comment by tjmoore** [comment]
It's not my PR, it's @svrooij 

I'd probably have to do my own PR unless I can add to this one, if I can work out the conflicts, unless @svrooij wants to fix the conflicts. I can test that it solves our issue.

**Comment by sherlock1982** [comment]
I will publish tomorrow. Glad this work was helpful :-)

File: Core/EwsHttpWebRequest.cs [hunk 1]
```diff
@@ -90,10 +90,12 @@ public async Task<IEwsHttpWebResponse> GetResponse(CancellationToken token)
...
-            message.Content.Headers.Clear();
-            message.Content.Headers.Add("Content-Type", "text/xml; charset=utf-8");
-            //if (!string.IsNullOrEmpty(ContentType))
-            //    message.Content.Headers.ContentType = new MediaTypeHeaderValue(ContentType);
+
+            if (!string.IsNullOrEmpty(ContentType)) {
+              message.Content.Headers.ContentType = null;
+              message.Content.Headers.TryAddWithoutValidation("Content-Type", ContentType);
+            }
+
```
Comment (tjmoore) [inline Core/EwsHttpWebRequest.cs:98]: Quick check. Is this the same effective fix as PR #52 ? (the change it replaces). That's where the merge conflict was I think as #32 was already fixed in 2.0.0-beta2.

**Pull request merged**